dbatools.psd1

```diff
@@ -540,6 +540,7 @@
         'Update-Dbatools',
         'Install-DbaWhoIsActive',
         'Install-DbaFirstResponderKit',
+        'Install-DbaMultiTool',
```

Comment:
Please reset this file. We will handle updating the manifest, more than our psd1 has to be udpated.